data/darktableconfig.xml.in

```diff
@@ -2737,7 +2737,7 @@
   </dtconfig>
   <dtconfig>
     <name>plugins/imageio/format/avif/color_mode</name>
-    <type min="0">int</type>
+    <type min="0" max="1">int</type>
```

Comment:
Looking at the code we can probably use a bool type here. Probably worth doing this now.

@kmilos : Is there a reason to have an `int` here?

Reply:
It's an enum in AVIF writer code currently. I don't know if more values are expected to be added...

Reply:
Sure it's an enum with 2 values:

```
enum avif_color_mode_e
{
  AVIF_COLOR_MODE_RGB = 0,
  AVIF_COLOR_MODE_GRAYSCALE,
};
```

And sadly inverted regarding the `color_mode` value. Yet, probably worth the effort now if you agree.

Reply:
> And sadly inverted regarding the `color_mode` value

I don't think so? We want "grayscale" to be 1/TRUE/"yes" in the genral case, and it seems to match current AVIF and TIFF writers behaviour well as is.

Sure, I can work on the bool conversion... In that case, you still want the dropbox, or shall we convert to checkbox just as well?

Reply:
A checkbox would be better I think. TIA.

Reply:
Btw, will this work w/ existing dartablerc? It seems to me that `dt_conf_get_bool()` will not return 1 if the old values was a string "1" (i.e. not "true" or "TRUE")?

Reply:
And it doesn't look like we have checkboxes available either?